recipes/onnxoptimizer/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "onnxoptimizer" %}
+{% set version = "0.3.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/onnxoptimizer-{{ version }}.tar.gz
+  sha256: e08b726e0d4577e51e529f36bc324bf11b7cff12852cf3eee081f05c8b8c6f33
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - python
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - onnx
```

Comment:
You will need a run dependency on the deployment target here. See the conda-forge docs on how to do it.